Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Apsitvarkom.Models;
+using Apsitvarkom.Models.DTO;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace Apsitvarkom.Api.Controllers  
+{
+    [ApiController]
+    [Route("/api/[controller]")]
+
+    public class PollutedLocationController : ControllerBase
+    {
+        /*temporary list PollutedLocations. Latter on we will have stored data in file or database. Then we will create
+        static DataContext object _context and in constructor
+        declare _context value to DataContext object
+        */
+        
+        private static IEnumerable<PollutedLocationDTO> PollutedLocations = new List<PollutedLocationDTO>
+        {
+
+            new PollutedLocationDTO
+            {
+                Id = "5be2354e-2500-4289-bbe2-66210592e17f",
+                Location = new LocationDTO
+                {
+                    Latitude = 12.1651,
+                    Longitude = 121.151,
+                },
+                Radius = 10,
+                Severity = "MODERATE",
+                Spotted  =  DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.ttt"),
+                Progress = 15,
+                Notes = "This is test notes"
+
+            }
+        };
+        /*@Parameters : Object Id 
+         * @Returns : List of objects that are still in List
+         * @Action : Deleted object from list
+         */
+
+        [HttpGet("{id}")]
+        public async Task<ActionResult<PollutedLocationDTO>> GetById(string id)
```

Comment:
You specify the method as async Task, but do not use await anywhere in the method. This conversion may be a new gradual increment to this task, consider using ```void```.